### PR TITLE
fix profile page image not clickable issue, fix settings page onboard…

### PR DESCRIPTION
**Pull request by verasunyy**
…ing page not direct back issue, fix navigation goBack button not go back issue

### CODE REVIEW

File: client/ios/STCSummer2019/Info.plist [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	<key>CFBundleDevelopmentRegion</key>
 	<string>en</string>
 	<key>CFBundleDisplayName</key>
-	<string>STCSummer2019</string>
+	<string>STC</string>
```
Comment (siddharthparmar7) [inline client/ios/STCSummer2019/Info.plist:8]: Did you change the project name?

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please resolve merge conflicts.

**Review by iyuvraajsingh — DISMISSED** [review]

File: client/ios/STCSummer2019/Info.plist (hunk 1 above)
Comment (verasunyy) [reply client/ios/STCSummer2019/Info.plist:8]: yes, I did.

File: client/js/components/ItemStoreCard/styles.js [hunk 2]
```diff
@@ -39,6 +39,34 @@ const styles = StyleSheet.create({
...
+    right: '5%',
+    top: -15,
+  },
+  partnerImage: {
+    width: '100%',
+    height: '100%',
+  },
+  partnerStoreCardContainer: {
+    width: '90%',
+    height: 100,
```
Comment (iyuvraajsingh) [inline client/js/components/ItemStoreCard/styles.js:56]: Is it possible to use %s here?

File: client/js/components/StoreTab/styles.js [hunk 3]
```diff
@@ -63,8 +63,8 @@ const styles = StyleSheet.create({
   },
   subfavItemWrapper: {
     width: '50%',
-    height: 235,
-    padding: 4,
+    height: 240,
+    // padding: 4,
```
Comment (iyuvraajsingh) [inline client/js/components/StoreTab/styles.js:67]: Please remove commented out code.

File: client/js/components/StoreTab/styles.js [hunk 4]
```diff
@@ -63,8 +63,8 @@ const styles = StyleSheet.create({
   },
   subfavItemWrapper: {
     width: '50%',
-    height: 235,
-    padding: 4,
+    height: 240,
```
Comment (iyuvraajsingh) [inline client/js/components/StoreTab/styles.js:66]: Is it possible to use %s here?

**Review by iyuvraajsingh — CHANGES_REQUESTED** [review]

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**